subt_ign/src/GameLogicPlugin.cc

```diff
@@ -1432,11 +1434,20 @@ bool GameLogicPluginPrivate::OnNewArtifact(const subt::msgs::Artifact &_req,
     return true;
   }
 
-  if (!this->finished && this->reportCount > this->reportCountLimit)
+  if (!this->finished && this->reportCount >= this->reportCountLimit)
   {
-    _resp.set_report_status("report limit exceeded");
-    this->Log() << "report_limit_exceeded" << std::endl;
-    ignmsg << "Report limit exceed." << std::endl;
+    _resp.set_report_status("report limit reached");
+    this->Log() << "report_limit_reached" << std::endl;
+    ignmsg << "Report limit reached." << std::endl;
+
+    std::ostringstream stream;
```

Comment:
This looks like it may duplicate logic from https://github.com/osrf/subt/pull/598/files#diff-8c34eb77e7a210f910eae0869c499eafL1380-L1387

Should the logging happen up there, and then here it will just `Finish` and `return True`?

Reply:
This logic catches when report limit is reached (not exceeded). The logic on 1378 should never actually happen now, but it's safe to leave in.